Doc/library/socket.rst

```diff
@@ -578,6 +578,20 @@ Other functions
 The :mod:`socket` module also offers various network-related services:
 
 
+.. function:: close(fd)
+
+   Close a socket file descriptor. This is like :func:`os.close`, but for
+   sockets. On some platforms (most noticeable Windows) :func:`os.close`
+   does not work for socket file descriptors.
+
+   .. versionadded:: 3.7
+
+.. function:: dup(fd)
+
+   Duplicate a socket file descriptor. This is like :func:`os.dup`, but for
+   sockets. On some platforms (most noticeable Windows) :func:`os.dup`
+   does not work for socket file descriptors.
+
```

Comment:
You should add a `versionadded` too.

Reply:
The socket.dup() function exists since 2.6 at least. It just hasn't been documented yet.